tests/integration/__init__.py

```diff
@@ -390,6 +390,17 @@ def _id_and_mark(f):
 requires_custom_payload = pytest.mark.skipif(SCYLLA_VERSION is not None or PROTOCOL_VERSION < 4,
                                             reason='Scylla does not support custom payloads. Cassandra requires native protocol v4.0+')
 xfail_scylla = lambda reason, *args, **kwargs: pytest.mark.xfail(SCYLLA_VERSION is not None, reason=reason, *args, **kwargs)
+
+
+def xfail_scylla_version(reason, oss_scylla_version, ent_scylla_version, *args, **kwargs):
```

Comment:
Maybe `xfail_scylla_version_less_than`? Current name doesn't properly convey the meaning.

Reply:
Yeah, I thought maybe to support both options somehow, but I didn't had time todo so.

Reply:
I think of putting  the version into  pytest_markeval_namespace so can use pytest.mark.xfail directly